src/thirdparty/gpacmp4/gpac/internal/isomedia_dev.h

```diff
@@ -509,7 +510,7 @@ typedef struct
 	tmp->type = __4cc;
 
 #define ISOM_DECREASE_SIZE(__ptr, bytes)	if (__ptr->size < (bytes) ) {\
-			GF_LOG(GF_LOG_ERROR, GF_LOG_CONTAINER, ("[isom] not enough bytes in box %s: %d left, reading %d (file %s, line %d)\n", gf_4cc_to_str(__ptr->type), __ptr->size, (bytes), __FILE__, __LINE__ )); \
+			GF_LOG(GF_LOG_ERROR, GF_LOG_CONTAINER, ("[isom] not enough bytes in box %s: %" PRIu64 " left, reading %d (file %s, line %d)\n", gf_4cc_to_str(__ptr->type), __ptr->size, (bytes), __FILE__, __LINE__ )); \
```

Comment:
Since this seems to be internal code from GPAC itself, did you verify this same issue isn't present on the upstream? If it's fixed there we might want to upgrade GPAC, if it's not, it might be more interesting to report it over there too.

Reply:
I haven't checked that yet but I can check right now.